tests/test_repository.py

```diff
@@ -186,14 +186,8 @@ def test_sle15_packages(container_per_test, pkg):
     )
 
 
-@pytest.mark.skipif(
```

Comment:
this part imho should not be reverted (unless the whole function is moved to all)

Reply:
Yes, done that now